uninstall.php

```diff
@@ -11,4 +11,22 @@
 	exit;
 }
 
-delete_option( 'perflab_modules_settings' );
+if ( is_multisite() && 100 >= get_sites( array( 'count' => true ) ) ) {
+
+	$site_ids = get_sites(
+		array(
+			'fields'                 => 'ids',
+			'number'                 => '100',
+			'update_site_cache'      => false,
+			'update_site_meta_cache' => false,
+		)
+	);
+
+	foreach ( $site_ids as $site_id ) {
+		switch_to_blog( $site_id );
+		delete_option( 'perflab_modules_settings' );
+	}
+	restore_current_blog();
+} else {
+	delete_option( 'perflab_modules_settings' );
+}
```

Comment:
I think it would be best to not have this in an `else`, but instead always run it. IMO it makes sense to at least always wipe the current site's (main site) option on uninstall. If a multisite has more than 100 sites, we should still ensure the main site's option is deleted.

So I would suggest the following:
```suggestion
}

// Delete the current site's option.
delete_option( 'perflab_modules_settings' );
```